kinematics/acceleration.py

```diff
@@ -0,0 +1,129 @@
+# KINEMATICS #
+
+# Kinematics is the physics of motion
+
+# Here is an explanation on the "big 4 equations", all of which were used to make this
+# algorithm: https://www.physicsclassroom.com/class/1dkin/Lesson-6/Kinematic-Equations
+
+# Output will always be a float, so if you need to round, set retaining_wall to
+# whatever place you are rounding to
+
+# NOTE: Not all of these equations will give you an exactly same answer, but will be
+# very close. Typically you would only show sig
+# figs when you calculate with these equations.
+
+# Doctests are all tested with initial_velocity=0, final_velocity=104.96,
+# initial_position=0, final_position=1720, acceleration=3.2,
+# time_elapsed=32.8, retaining_wall=2
+
+# These were calculated by hand so the output for distance is different, but the outputs
+# are more accurate than the one used for the example.
+
+# In all cases distance is final_position-initial_position
+# TOTAL distance is final_position+initial_position
+# If you want to find distance from one point to another assume initial_position=0
+
+# Typing hints
```

Comment:
We can lose the comment which is redundant.  Please add
```suggestion
from __future__ import annotations

```
so that `pyupgrade` can replace all `Optional` and `Union` with modern syntax.  Same changes please on the other files.